helm/prometheus-operator-app/templates/networkpolicy-admission.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and  (index .Values "prometheus-operator-app.prometheusOperator.enabled") (index .Values "prometheus-operator-app.prometheusOperator.admissionWebhooks.enabled") (index .Values "prometheus-operator-app.prometheusOperator.admissionWebhooks.patch.enabled") (not (index .Values "prometheus-operator-app.prometheusOperator.admissionWebhooks.certManager.enabled")) }}
+{{- if and  (index .Values "prometheus-operator-app" "prometheusOperator" "enabled") (index .Values "prometheus-operator-app" "prometheusOperator" "admissionWebhooks" "enabled") (index .Values "prometheus-operator-app" "prometheusOperator" "admissionWebhooks" "patch" "enabled") (not (index .Values "prometheus-operator-app" "prometheusOperator" "admissionWebhooks" "certManager" "enabled")) }}
```

Comment:
Why don't we keep the previous dotted annotation? It was easier to read.

Reply:
That doesn’t work